### PR TITLE
Add definition for module

### PR DESCRIPTION
**Pull request by RexJaeschke**
This addresses Issue https://github.com/dotnet/csharpstandard/issues/582.

### CODE REVIEW

File: standard/terms-and-definitions.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ For the purposes of this specification, the following definitions apply. Other t
   - exceptional condition reported during program execution
 - **implementation**
   - particular set of software (running in a particular translation environment under particular control options) that performs translation of programs for, and supports execution of methods in, a particular execution environment
+- **module**
+  - the contents of an assembly produced by a compiler.  Some implementations may have facilities to produce assemblies that contain more than one module.  The behavior in such situations is outside the scope of this specification
```
Comment (jskeet) [inline standard/terms-and-definitions.md:38]: Add a period at the end given that it's a third complete sentence?

**Review by jskeet — APPROVED** [review]
Basically fine. (Personally I don't like double spaces, but it's no big deal.)

File: standard/terms-and-definitions.md (hunk 1 above)
Comment (RexJaeschke) [reply standard/terms-and-definitions.md:38]: @jskeet None of the entries in this chapter are terminated with periods, so I was being consistent. In any event, the style of each entry was initially based on ISO rules, which require that each definition be able to replace directly an instance of the term. Of course, that doesn't work grammatically if a definition contains multiple sentences. So, for now, I'll leave it as is.

**Pull request merged**